source/js/foundation/template-js-handler/index.js

```diff
@@ -11,7 +11,7 @@ import publicationSummaryBar from "./publication-summary-bar.js";
 import continuousScrollingBox from "./continuous-scrolling-box.js";
 import pulseProfileCardHandler from "./pulse-profile-card.js";
 import pulseProfileListFilterHandler from "./pulse-profile-list-filter.js";
-import articleHeroVideoHandler from "./article-hero-video-handler.js";
+import publicationArticleHeroVideoHandler from "./publication-article-hero-video-handler.js";
```

Comment:
Some updates due to the name change of the file

Reply:
**Suggestion** Since you already named the hero template to "custom_hero" it would be more consistent and less confusing to call this "custom-hero-video-handler", in my opinion.